ark/phenotyping/cell_cluster_utils_test.py

```diff
@@ -11,7 +11,7 @@
 from matplotlib.colors import ListedColormap
 import numpy as np
 import pandas as pd
-from skimage.draw import circle
+from skimage.draw import disk
```

Comment:
Is this import used anywhere?

Reply:
Nope. Will remove it. Should I go through and remove all the other unused import statements?

Reply:
Open an issue, we can do it all at once after the workshop as a cleanup

Reply:
Sounds good.